src/main/java/org/jboss/ejb/client/DiscoveryEJBClientInterceptor.java

```diff
@@ -340,6 +348,7 @@ Discovery getDiscovery() {
             final Map.Entry<URI, String> entry = nodes.entrySet().iterator().next();
             location = entry.getKey();
             nodeName = entry.getValue();
+            Logs.INVOCATION.tracef("Performed first-match discovery(target affinity = %s, destination = %s)", new NodeAffinity(nodeName), location);
```

Comment:
This should just log the node name so that we don't pay the construction cost when trace is disabled.